[2024-12-06T19:13:39.118+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T19:09:34.321126+00:00 [queued]>
[2024-12-06T19:13:39.128+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T19:09:34.321126+00:00 [queued]>
[2024-12-06T19:13:39.129+0000] {taskinstance.py:2170} INFO - Starting attempt 3 of 3
[2024-12-06T19:13:39.145+0000] {taskinstance.py:2191} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2024-12-06 19:09:34.321126+00:00
[2024-12-06T19:13:39.152+0000] {standard_task_runner.py:60} INFO - Started process 1271 to run task
[2024-12-06T19:13:39.157+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2024-12-06T19:09:34.321126+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpwz3ur9gt']
[2024-12-06T19:13:39.158+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask is_weather_api_ready
[2024-12-06T19:13:39.171+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-06T19:13:39.210+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2024-12-06T19:09:34.321126+00:00 [running]> on host 8a5e48d7ee4d
[2024-12-06T19:13:39.272+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T19:09:34.321126+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-06T19:09:34.321126+00:00'
[2024-12-06T19:13:39.273+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:13:39.286+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:13:43.267+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:13:43.268+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:14:43.330+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:14:43.335+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:14:48.231+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:14:48.232+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:15:47.601+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:15:47.612+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:15:51.637+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:15:51.637+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:16:51.699+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:16:51.711+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:16:56.590+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:16:56.590+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:17:56.625+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:17:56.632+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:17:57.409+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:17:57.409+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:18:57.469+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:18:57.480+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:18:58.274+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:18:58.275+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:19:58.337+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:19:58.345+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:20:01.522+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:20:01.523+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:21:01.585+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:21:01.592+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:21:05.588+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:21:05.589+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:22:05.651+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:22:05.661+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:22:07.603+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:22:07.604+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:23:07.666+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:23:07.673+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:23:08.467+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:23:08.468+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:24:08.530+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:24:08.544+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:24:10.164+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:24:10.164+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:25:10.226+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:25:10.242+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:25:13.521+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:25:13.521+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:26:13.537+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:26:13.551+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:26:15.539+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:26:15.540+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:27:15.601+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:27:15.615+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:27:19.538+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:27:19.538+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:28:19.600+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:28:19.613+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:28:21.362+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:28:21.363+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:29:21.400+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:29:21.406+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:29:21.587+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:29:21.587+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:30:21.599+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:30:21.611+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:30:22.186+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:30:22.186+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:31:22.222+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:31:22.228+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:31:23.803+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:31:23.804+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:32:23.829+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:32:23.835+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:32:24.028+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:32:24.029+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:33:24.091+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:33:24.102+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:33:24.795+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:33:24.796+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:34:24.857+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:34:24.864+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:34:26.044+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:34:26.044+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:35:26.065+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:35:26.079+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:35:26.578+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:35:26.579+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:36:26.641+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:36:26.654+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:36:30.065+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:36:30.065+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:37:30.127+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:37:30.138+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:37:35.615+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:37:35.615+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:38:35.636+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:38:35.646+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:38:41.523+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:38:41.524+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:39:41.094+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:39:41.104+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:39:43.045+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:39:43.045+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:40:43.107+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:40:43.113+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:40:43.836+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:40:43.836+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:41:43.897+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:41:43.910+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:41:44.324+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:41:44.325+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:42:44.388+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:42:44.394+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:42:45.787+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:42:45.788+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:43:45.850+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:43:45.858+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:43:50.019+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:43:50.019+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:44:50.076+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:44:50.088+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:44:53.129+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:44:53.130+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:45:53.144+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:45:53.155+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:45:56.293+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:45:56.294+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:46:56.356+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:46:56.363+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:46:59.845+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:46:59.846+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:47:59.908+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:47:59.914+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:48:00.420+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:48:00.421+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:49:00.471+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:49:00.479+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:49:01.446+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:49:01.446+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:50:01.508+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:50:01.520+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:50:04.203+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:50:04.203+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:51:04.248+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:51:04.260+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:51:05.915+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:51:05.916+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:52:05.977+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:52:05.989+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:52:09.269+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:52:09.270+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:53:09.333+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:53:09.342+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:53:14.829+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:53:14.830+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:54:14.892+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:54:14.903+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:54:15.149+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:54:15.150+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:55:15.212+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:55:15.221+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:55:19.045+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:55:19.045+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:56:19.107+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:56:19.119+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:56:20.805+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:56:20.805+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:57:20.844+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:57:20.852+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:57:21.115+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:57:21.115+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:58:21.176+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:58:21.188+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:58:24.104+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:58:24.105+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T19:59:24.160+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T19:59:24.165+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T19:59:26.341+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T19:59:26.342+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:00:26.349+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:00:26.360+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:00:29.196+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:00:29.196+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:01:29.254+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:01:29.261+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:01:29.777+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:01:29.777+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:02:29.838+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:02:29.850+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:02:33.733+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:02:33.733+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:03:33.795+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:03:33.801+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:03:36.909+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:03:36.909+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:04:36.971+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:04:36.983+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:04:39.045+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:04:39.045+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:05:39.072+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:05:39.079+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:05:45.016+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:05:45.017+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:06:45.079+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:06:45.090+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:06:47.043+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:06:47.043+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:07:47.104+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:07:47.112+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:07:49.316+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:07:49.317+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:08:49.378+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:08:49.388+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:08:52.307+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:08:52.307+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:09:52.368+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:09:52.382+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:09:53.036+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:09:53.037+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:10:53.099+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:10:53.111+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:10:55.043+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:10:55.043+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:11:54.561+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:11:54.571+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:11:55.645+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:11:55.645+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:12:55.688+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:12:55.698+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:12:59.658+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:12:59.659+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:13:59.719+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:13:59.728+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:13:59.913+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:13:59.914+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:15:00.444+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:15:00.452+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:15:01.708+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:15:01.709+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:16:01.771+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:16:01.778+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:16:04.130+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:16:04.131+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:17:04.136+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:17:04.146+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:17:04.369+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:17:04.370+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:18:04.432+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:18:04.442+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:18:08.971+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:18:08.971+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:19:09.033+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:19:09.040+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:19:11.291+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:19:11.291+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:20:11.353+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:20:11.367+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:20:12.467+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:20:12.467+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:21:12.529+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:21:12.535+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:21:15.090+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:21:15.090+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:22:15.152+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:22:15.162+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:22:20.938+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:22:20.939+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:23:20.984+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:23:20.991+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:23:21.283+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:23:21.283+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:24:21.332+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:24:21.348+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:24:22.732+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:24:22.732+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:25:22.793+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:25:22.798+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:25:25.005+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:25:25.006+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:26:25.068+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:26:25.077+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:26:26.251+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:26:26.251+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:27:26.312+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:27:26.324+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:27:26.532+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:27:26.533+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:28:26.589+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:28:26.598+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:28:28.786+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:28:28.787+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:29:28.849+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:29:28.855+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:29:32.802+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:29:32.802+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:30:32.830+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:30:32.840+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:30:33.428+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:30:33.429+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:31:33.491+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:31:33.500+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:31:36.844+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:31:36.844+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:32:36.905+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:32:36.914+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:32:37.415+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:32:37.416+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:33:37.477+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:33:37.490+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:33:38.544+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:33:38.545+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:34:38.579+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:34:38.591+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:34:41.037+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:34:41.037+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:35:41.099+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:35:41.106+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:35:42.093+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:35:42.094+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:36:42.156+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:36:42.167+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:36:44.747+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:36:44.747+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:37:44.808+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:37:44.820+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:37:46.956+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:37:46.957+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:38:46.992+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:38:47.003+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:38:48.843+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:38:48.843+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:39:48.904+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:39:48.915+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:39:50.987+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:39:50.987+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:40:51.049+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:40:51.062+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:40:53.514+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:40:53.515+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:41:53.577+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:41:53.581+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:41:56.185+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:41:56.185+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:42:56.247+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:42:56.259+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:43:01.003+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:43:01.004+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:44:01.026+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:44:01.037+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:44:05.004+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:44:05.004+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:45:05.068+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:45:05.074+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:45:05.336+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:45:05.336+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:46:05.377+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:46:05.389+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:46:06.293+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:46:06.294+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:47:06.346+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:47:06.358+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:47:06.827+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:47:06.828+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:48:06.889+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:48:06.897+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:48:08.844+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:48:08.845+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:49:08.906+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:49:08.917+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:49:09.648+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:49:09.649+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:50:09.711+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:50:09.717+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:50:11.589+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:50:11.589+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:51:11.624+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:51:11.631+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:51:11.826+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:51:11.827+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:52:11.890+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:52:11.901+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:52:12.523+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:52:12.524+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:53:12.585+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:53:12.590+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:53:12.747+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:53:12.748+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:54:12.808+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:54:12.819+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:54:13.149+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:54:13.149+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:55:13.210+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:55:13.227+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:55:13.458+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:55:13.459+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:56:13.521+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:56:13.530+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:56:13.948+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:56:13.949+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:57:14.012+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:57:14.025+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:57:14.341+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:57:14.342+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:58:14.380+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:58:14.392+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:58:15.443+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:58:15.444+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T20:59:15.505+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T20:59:15.513+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T20:59:16.875+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T20:59:16.875+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:00:16.936+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:00:16.947+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:00:19.020+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:00:19.021+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:01:19.084+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:01:19.090+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:01:23.473+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:01:23.474+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:02:23.513+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:02:23.527+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:02:23.744+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:02:23.744+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:03:23.768+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:03:23.780+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:03:25.925+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:03:25.926+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:04:25.988+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:04:25.995+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:04:26.748+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:04:26.749+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:05:26.812+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:05:26.824+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:05:28.524+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:05:28.525+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:06:28.588+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:06:28.595+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:06:32.972+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:06:32.972+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:07:33.034+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:07:33.047+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:07:38.237+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:07:38.237+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:08:38.299+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:08:38.311+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:08:42.049+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:08:42.049+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:09:42.110+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:09:42.117+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:09:44.779+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:09:44.779+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:10:44.841+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:10:44.852+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:10:46.552+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:10:46.553+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:11:46.559+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:11:46.571+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:11:50.603+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:11:50.603+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:12:50.632+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:12:50.643+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:12:52.248+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:12:52.249+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:13:52.306+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:13:52.312+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:13:53.549+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:13:53.549+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:14:53.611+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:14:53.619+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:14:54.107+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:14:54.107+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:15:54.168+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:15:54.181+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:15:57.003+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:15:57.003+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:16:57.022+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:16:57.033+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:16:57.642+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:16:57.642+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:17:57.704+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:17:57.711+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:17:59.649+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:17:59.649+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:18:59.710+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:18:59.718+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:19:01.836+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:19:01.837+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:20:01.900+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:20:01.913+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:20:05.004+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:20:05.005+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:21:05.067+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:21:05.078+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:21:06.694+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:21:06.694+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:22:06.740+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:22:06.754+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:22:07.675+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:22:07.676+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:23:07.725+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:23:07.732+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:23:09.003+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:23:09.003+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:24:09.059+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:24:09.071+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:24:12.312+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:24:12.312+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:25:12.355+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:25:12.367+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:25:14.452+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:25:14.452+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:26:14.475+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:26:14.486+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:26:18.123+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:26:18.123+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:27:18.158+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:27:18.170+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:27:21.099+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:27:21.099+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:28:21.121+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:28:21.128+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:28:22.987+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:28:22.987+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:29:23.570+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:29:23.582+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:29:25.461+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:29:25.462+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:30:25.524+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:30:25.532+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:30:27.629+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:30:27.630+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:31:27.692+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:31:27.704+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:31:29.781+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:31:29.781+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:32:29.842+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:32:29.852+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:32:32.189+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:32:32.190+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:33:32.252+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:33:32.263+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:33:33.460+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:33:33.461+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:34:33.522+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:34:33.529+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:34:34.640+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:34:34.641+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:35:34.702+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:35:34.712+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:35:39.646+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:35:39.647+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:36:39.708+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:36:39.722+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:36:40.597+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:36:40.598+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:37:40.659+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:37:40.665+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:37:41.523+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:37:41.524+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:38:41.137+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:38:41.150+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:38:45.217+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:38:45.217+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:39:45.271+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:39:45.277+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:39:46.966+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:39:46.967+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:40:47.028+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:40:47.039+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:40:49.299+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:40:49.300+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:41:49.360+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:41:49.367+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:41:49.547+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:41:49.548+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:42:49.603+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:42:49.611+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:42:52.358+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:42:52.358+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:43:52.420+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:43:52.432+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:43:52.645+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:43:52.646+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:44:52.662+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:44:52.669+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:44:56.700+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:44:56.700+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:45:56.762+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:45:56.770+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:45:59.150+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:45:59.151+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:46:59.184+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:46:59.191+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:47:01.845+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:47:01.846+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:48:01.908+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:48:01.920+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:48:06.101+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:48:06.101+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:49:06.163+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:49:06.169+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:49:07.782+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:49:07.783+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:50:07.844+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:50:07.851+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:50:11.664+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:50:11.664+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:51:11.691+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:51:11.697+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:51:13.301+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:51:13.302+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:52:13.363+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:52:13.374+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:52:14.102+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:52:14.103+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:53:14.164+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:53:14.179+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:53:15.030+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:53:15.030+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:54:15.092+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:54:15.107+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:54:15.524+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:54:15.524+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:55:15.586+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:55:15.596+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:55:17.332+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:55:17.332+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:56:17.394+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:56:17.405+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:56:19.028+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:56:19.029+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:57:19.090+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:57:19.097+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:57:21.108+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:57:21.109+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:58:21.169+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:58:21.182+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:58:25.044+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:58:25.044+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T21:59:25.104+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T21:59:25.115+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T21:59:27.060+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T21:59:27.060+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:00:27.123+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:00:27.134+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:00:27.805+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:00:27.806+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:01:27.866+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:01:27.878+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:01:32.887+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:01:32.887+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:02:32.946+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:02:32.956+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:02:33.653+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:02:33.654+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:03:33.715+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:03:33.732+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:03:37.656+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:03:37.656+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:04:37.718+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:04:37.724+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:04:41.045+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:04:41.045+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:05:41.107+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:05:41.117+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:05:41.432+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:05:41.433+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:06:41.496+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:06:41.502+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:06:45.828+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:06:45.829+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:07:45.840+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:07:45.851+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:07:46.999+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:07:47.000+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:08:47.061+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:08:47.067+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:08:47.388+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:08:47.389+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:09:47.450+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:09:47.463+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:09:47.681+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:09:47.682+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:10:47.737+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:10:47.749+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:10:49.300+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:10:49.301+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:11:49.362+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:11:49.370+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:11:50.001+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:11:50.001+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:12:50.062+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:12:50.073+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:12:50.830+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:12:50.831+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:13:50.891+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:13:50.897+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:13:52.560+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:13:52.561+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:14:52.622+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:14:52.628+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:14:52.853+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:14:52.853+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:15:52.914+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:15:52.925+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:15:55.094+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:15:55.095+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:16:55.156+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:16:55.163+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:16:56.013+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:16:56.014+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:17:56.077+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:17:56.089+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:17:56.910+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:17:56.910+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:18:56.972+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:18:56.984+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:19:00.117+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:19:00.117+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:20:00.178+0000] {http.py:130} INFO - Poking: https://api.openweathermap.org/data/2.5/weather?q=Nantes&appid=92a2228961d1701676a43daabb355f4d
[2024-12-06T22:20:00.185+0000] {base.py:83} INFO - Using connection ID 'weathermap_api' for task execution.
[2024-12-06T22:20:02.900+0000] {http.py:178} ERROR - HTTP error: Not Found
[2024-12-06T22:20:02.900+0000] {http.py:179} ERROR - <html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

[2024-12-06T22:20:50.668+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to removed. Terminating instance.
[2024-12-06T22:20:50.670+0000] {process_utils.py:131} INFO - Sending 15 to group 1271. PIDs of all processes in the group: [1271]
[2024-12-06T22:20:50.670+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 1271
[2024-12-06T22:20:50.670+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-06T22:20:50.722+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=1271, status='terminated', exitcode=0, started='19:13:38') (1271) terminated with exit code 0
